shaders/patch_control_points/tess.vert

```diff
@@ -35,16 +31,13 @@ layout(push_constant) uniform Push_Constants
 }
 push_constants;
 
-
 void main(void)
 {
-	mat4 rotX = mat4(1.0f);
-	rotX[1][1] = cos(PI);
-	rotX[1][2] = sin(PI);
-	rotX[2][1] = -sin(PI);
-	rotX[2][2] = cos(PI);
-	vec4 pos = rotX * vec4(inPos + push_constants.direction, 1.0f);
+	mat4 rotX   = mat4(1.0f);
+	rotX[1][1]  = cos(PI);
+	rotX[1][2]  = sin(PI);
+	rotX[2][1]  = -sin(PI);
+	rotX[2][2]  = cos(PI);
+	vec4 pos    = rotX * vec4(inPos + push_constants.direction, 1.0f);
```

Comment:
skip `pos` and directly assign the result to `gl_Position`?

Reply:
Fixed